<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTorrentFirstStep</class>
 <widget class="QWidget" name="NewTorrentFirstStep">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main parameters</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/resources/images/bittorrent.svg</normaloff>:/resources/images/bittorrent.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Output:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLineEdit" name="Output_"/>
     </item>
     <item>
      <widget class="QPushButton" name="BrowseOutput_">
       <property name="text">
        <string>Browse...</string>
       </property>
       <property name="icon">
        <iconset theme="document-open-folder"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Announce URL:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="AnnounceURL_"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Torrent's root path:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="RootPath_"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="BrowseFile_">
         <property name="text">
          <string>File...</string>
         </property>
         <property name="icon">
          <iconset theme="document-open"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BrowseDirectory_">
         <property name="text">
          <string>Directory...</string>
         </property>
         <property name="icon">
          <iconset theme="document-open-folder"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Date:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDateEdit" name="Date_">
     <property name="displayFormat">
      <string>dd MMM yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Comment:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1" rowspan="2">
    <widget class="QTextEdit" name="Comment_"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
